packages/aws-cdk-lib/aws-secretsmanager/lib/rotation-schedule.ts

```diff
@@ -37,7 +37,7 @@ export interface RotationScheduleOptions {
    * Specifies the number of days after the previous rotation before
    * Secrets Manager triggers the next automatic rotation.
    *
```

Comment:
```suggestion
   *
   * The maximum value is 1000 days.
   *
```
Documentation about accepted values.

Reply:
https://github.com/aws/aws-cdk/pull/27570/commits/eb2392dcb97be0ebbd22f30ed5bf056dad112e24

Reply:
https://github.com/aws/aws-cdk/pull/27592/files